lib/page/house_wikidata.rb

```diff
@@ -12,21 +13,49 @@ def title
       "EveryPolitician: #{country.name} — #{house.name}"
     end
 
-    def country
-      index.country(country_slug)
+    def house_name
+      house.name
     end
 
-    def house
-      country.legislature(house_slug)
+    def country
+      index.country(country_slug)
     end
 
     # TODO: we shouldn't be passing raw Popolo, only what's needed
     def popolo
       house.popolo
     end
 
+    def people_with_wikidata
+      result = popolo.persons.reduce([]) do |arr, person|
+        if scheme = person.identifiers.select { |p| p[:scheme] == 'wikidata' }
+          wikidata_identifier = scheme.first[:identifier]
+          wikidata_url = "https://www.wikidata.org/wiki/#{wikidata_identifier}"
+          arr.push( { name: person.name,
+                      wikidata_identifier: wikidata_identifier,
+                      wikidata_url: wikidata_url
+                    }
```

Comment:
I don't think we need to collapse this down to our own data structure here — it's fine just to pass in a list of `EveryPolitician::Person` objects